core/src/components/toggle/toggle.scss

```diff
@@ -11,6 +11,8 @@
   }
 
   input[type='checkbox'] {
+    appearance: initial; /* Firefox */
+    appearance: initial; /* Safari/Chrome */
```

Comment:
Is it needed twice?

Reply:
Not sure what happened here.. commit incoming